callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java

```diff
@@ -139,45 +147,49 @@ public void run() {
 	/**
 	 * Send keep alive bridge message.
 	 *
-	 * @param appId the app id            
-	 * @param bridgeId the bridge id         
-	 * @param token the token           
-	 * @param httpClient the http client
-	 *            
+	 * @param appId
+	 *            the app id
+	 * @param bridgeId
+	 *            the bridge id
+	 * @param token
+	 *            the token
+	 * @param httpClient
+	 *            the http client
+	 * 
 	 */
 	private void sendKeepAliveBridgeMessage(int appId, String bridgeId, String token, final CallStatsHttp2Client httpClient) {
 		long apiTS = System.currentTimeMillis();
-		BridgeKeepAliveMessage message = new BridgeKeepAliveMessage(appId, bridgeId, CallStatsConst.CS_VERSION, apiTS, token);
```

Comment:
Do we send the version somewhere else?

Reply:
may be, need consultation with @stenhagglund